### PR TITLE
New TC for [PolicyTableUpdate] Policy Manager responds on GetURLs fro…

### PR DESCRIPTION
**Pull request by istoimenova**
To run the script you need to replace
user_modules/shared_testcases/testCasesForPolicyTable.lua
user_modules/shared_testcases/testCasesForPolicyTableSnapshot.lua
user_modules/shared_testcases/testCasesForBuildingSDLPolicyFlag.lua
from #32. Common functions are under review.

@maria-mih please review the script according to template.

### CODE REVIEW

File: test_scripts/Polices/Policy_Table_Update/009_ATF_P_TC_PTU_GetURLs.lua [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+config.defaultProtocolVersion = 2
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+local mobile_session = require('mobile_session')
+
+--[[ Preconditions ]]
+testCasesForPolicyTable:flow_PTU_SUCCEESS_EXTERNAL_PROPRIETARY()
```
Comment (istoimenova) [inline test_scripts/Polices/Policy_Table_Update/009_ATF_P_TC_PTU_GetURLs.lua]: @istoimenova Call of function should be updated to be included in Test.

File: test_scripts/Polices/Policy_Table_Update/009_ATF_P_TC_PTU_GetURLs.lua [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+-- Request PTU
+testCasesForPolicyTable:trigger_PTU_user_request_update_from_HMI()
```
Comment (istoimenova) [inline test_scripts/Polices/Policy_Table_Update/009_ATF_P_TC_PTU_GetURLs.lua]: @istoimenova Call of function should be updated to be included in Test.

File: test_scripts/Polices/Policy_Table_Update/009_ATF_P_TC_PTU_GetURLs.lua [hunk 3]
```diff
@@ -0,0 +1,78 @@
+---------------------------------------------------------------------------------------------
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/009_ATF_P_TC_PTU_GetURLs.lua]: Review of template

File: test_scripts/Polices/Policy_Table_Update/009_ATF_P_TC_PTU_GetURLs.lua [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+-- Requirements summary:
+-- [PolicyTableUpdate] Policy Manager responds on GetURLs from HMI
+-- In case HMI sends GetURLs and at least one app is registered SDL must return only default url and url related to registered app
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/009_ATF_P_TC_PTU_GetURLs.lua]: Shouldn't this be moved to section Description?

File: test_scripts/Polices/Policy_Table_Update/009_ATF_P_TC_PTU_GetURLs.lua [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+-- [HMI API] GetURLs request/response
+--
+-- Description:
+-- SDL should request PTU in case user requests PTU
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/009_ATF_P_TC_PTU_GetURLs.lua]: Please check description appropriateness.

File: test_scripts/Polices/Policy_Table_Update/009_ATF_P_TC_PTU_GetURLs.lua [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+  EXPECT_HMIRESPONSE(RequestId,{result = {code = 0, method = "SDL.GetURLS", urls = endpoints} } )
+  :Do(function(_,data) 
+  	if(is_app_esxist == false) then
+  	  self:FailTestCase("endpoints for application doesn't exist!")
+    end
+  end)
+end
+
+return Test
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/009_ATF_P_TC_PTU_GetURLs.lua]: Maybe need to add Postconditions section?

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge.

**Pull request merged**